app/controllers/api/functions.php

```diff
@@ -938,7 +938,7 @@
         ]);
 
         /** Execute function */
-        $executor = new Executor();
+        $executor = new Executor(App::getEnv('_APP_EXECUTOR_HOST', 'http://appwrite-executor/v1'));
```

Comment:
Let's keep the default value set from env vars, and leave the codebase cleaner for hard-coded values.